src/test/msgr/test_msgr.cc

```diff
@@ -22,6 +22,14 @@
 #include <time.h>
 #include <set>
 #include <list>
+
+#include <boost/random/mersenne_twister.hpp>
+#include <boost/random/uniform_int.hpp>
+#include <boost/random/binomial_distribution.hpp>
```

Comment:
Might want to order these includes in the alphabetical order.

Reply:
Will do. Thanks for pointing this out!